tests/scripts/set_up_end_to_end_tests_env.sh

```diff
@@ -17,6 +17,7 @@ set_up_end_to_end_tests_env() {
   source env/bin/activate
 
   python -m pip config set global.extra-index-url https://download.pytorch.org/whl/cpu
+  python -m pip install setuptools wheel
```

Comment:
Related to https://github.com/tenstorrent/tt-metal/pull/11870/files#r1731731907

Reply:
Related to my question above. What did you see that led you to add this?